### PR TITLE
Add a new game, Simonionburg

### PR DESCRIPTION
**Pull request by SimonW-Game**
A game made by *Simon* based on Domin*ion* and Quacks of Quedlin*burg* (crafty, I know).

Hi, the name is Simon.  New to node, not to code.  Big fan of Rocketcrab.  Made a game.  Here it is.  Will continue making improvements to it (as well as basic enhancements like minification and such).  ~I will also be finishing the rest of the rules tomorrow.~ [EDIT: Rules are finished]

It's a more difficult game to learn, but it plays extremely similarly to Quacks of Quendlinburg (if you've played that) so someone should be able to pick it up if they know that game.

### CODE REVIEW

File: config/games/simonionburg.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { randomBytes } from "crypto";
+
+const game: ServerGame = {
+	id: "simonionburg",
+	name: "Simonionburg",
+	author: "Simon Weisse",
+	basedOn: {
+		game: "Quacks of Quedlinburg and Dominion (losely)",
+		author: "Wolfgang Warsch (Quacks), Donald X. Vaccarino (Dominion)",
+		link: "https://boardgamegeek.com/boardgame/244521/quacks-quedlinburg", // https://dominion.games/ for a second game link /shrug
```
Comment (SimonW-Game) [inline config/games/simonionburg.ts:11]: I don't think it's common to base a game off of two games, but not being able to link both of them is a conundrum.  Not a big deal, but thought I'd mention it.

File: config/games/simonionburg.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { ServerGame } from "../../types/types";
+import { randomBytes } from "crypto";
+
+const game: ServerGame = {
+	id: "simonionburg",
+	name: "Simonionburg",
+	author: "Simon Weisse",
+	basedOn: {
+		game: "Quacks of Quedlinburg and Dominion (losely)",
```
Comment (alenros) [inline config/games/simonionburg.ts]: There's a typo in loosely

**Comment by alenros** [comment]
This is brilliant!
Need help with the development?

**Comment by SimonW-Game** [comment]
> This is brilliant!
> 
> Need help with the development?

Thanks! I'll let you know! I gotta make sure my GitHub repo is setup properly. I've used gitlab and bitbucket, but never GitHub.

**Comment by SimonW-Game** [comment]
Free azure subscription ran out this morning. Will fix tonight and get the minifixation out there too. (I thought I had until 2/1, guess not). 

EDIT: Minification done ✅

**Comment by SimonW-Game** [comment]
Anything I should be doing before this can be merged?

**Comment by tannerkrewson** [comment]
hey i'm not ignoring you. i'm just a really bad procrastinator. will get around to reviewing this at some point. sorry for the delay 🙂

**Comment by SimonW-Game** [comment]
Ha, no problem! I just wasn't sure if it was an issue on my end.

**Comment by SimonW-Game** [comment]
Any news on this?

**Comment by tannerkrewson** [comment]
Sorry for the wait! Will get to it eventually ! Feel free to keep pestering me I deserve it 😅

**Comment by SimonW-Game** [comment]
> Sorry for the wait! Will get to it eventually ! Feel free to keep pestering me I deserve it 😅 

Well, I can work with you if that would make it easier.

**Comment by SimonW-Game** [comment]
Can I assume this will not be added?  It's alright if you don't like it or don't think it will fit your site. I would just like to know.

**Comment by tannerkrewson** [comment]
Hey! Honestly I haven't even looked at any of the games in any of the PRs
yet. Sorry that's it's taken so long; it's unacceptable of me. I will get
to it as soon as I can!

**Comment by tannerkrewson** [comment]
I appreciate that you opened the PR in the first place, and apologies for never responding to this. 

The game link looks dead, so closing out for now. Please reopen if you'd like to try again!

**Pull request closed without merging**